chapters/05-version-control-solo.Rmd

```diff
@@ -23,7 +132,6 @@ TODO: This question makes a change to zipfs, so this should be reflected in any
 
 1. For the `zipfs` previously used for demonstration in this chapter,
 perform the following tasks:
-    - Initialize the repository (`use_git()` in the R Console).
     - Open up a Terminal and create a new branch called `new-feature` with `git`.
```

Comment:
We might need to change this over if we decide on using gert instead of Terminal.